docs/index.md

```diff
@@ -353,10 +353,16 @@ The following limitations apply when presenting Health Card as QR codes, rather
 
 ### Numerical Encoding
 
-When printing or displaying a Health Card as a QR code, the the JWS string value SHALL be represented as a Numeric Mode value consisting of the characters `0`-`9`. Each character "c" of the JWS is converted into a sequence of two digits as by taking `Ord(c)-45` and treating the result as a two-digit base ten number. For example, `'X'` is encoded as `43`, since `Ord('X')` is `88`, and `88-45` is `43`. (The constant "45" appears here because it is the ordinal value of `-`, the lowest-valued character that can appear in a compact JWS. Subtracting 45 from the ordinal values of valid JWS characters produces a range between 00 and 99, ensuring that each character of the JWS can be represented in exactly two base-10 numeric digits.)
+When printing or displaying a Health Card as a QR code, the the JWS string value SHALL be represented as two segments:
+
+1. An `alphanumeric` mode segment consisting of the fixed string `SHC://`
```

Comment:
I assume the intent is for the QR code payload to be a URI. If so, the scheme is case-insensitive (https://tools.ietf.org/html/rfc3986#section-3.1) so it might be worth calling that out so verifier don't get tripped up. I would also prefer using lower-case here since it's more common.

On a related note, what's the purpose of the `\\`? Wouldn't csh:DIGITS be simpler, as is done with `data:` URI?

Reply:
Agree, I don't think we need '//'? Also, uppercase or lowercase scheme?

Reply:
On mobile platforms, are apps allowed to register for custom schemes without `://`? Are URIs recognized as routable? My experience was no, but I may be missing something.

Reply:
> I would also prefer using lower-case here since it's more common.

This isn't part of the alphanumeric character set in QR codes, right?

Reply:
> I don't think we need '//'

* iOS: https://developer.apple.com/documentation/xcode/allowing_apps_and_websites_to_link_to_your_content/defining_a_custom_url_scheme_for_your_app indicates that `customscheme:value` URIs should be routable / associable with apps
* https://developer.android.com/guide/topics/manifest/data-element#scheme indicates  the same, though also has the note below:

> Note: Scheme matching in the Android framework is case-sensitive, unlike the RFC. As a result, you should always specify schemes using lowercase letters.

Reply:
Oooh you're right! You register the scheme alone, but the route is required.

Reply:
I'll make updates per these suggestions.

Reply:
On further discussion, lower case is recommended for Android and using `//` is required for browser redirects (at least) on iOS.